content/actions/reference/workflow-commands-for-github-actions.md

```diff
@@ -210,22 +210,36 @@ echo "::add-mask::$MY_NAME"
 
 Stops processing any workflow commands. This special command allows you to log anything without accidentally running a workflow command. For example, you could stop logging to output an entire script that has comments.
 
-### Example stopping workflow commands
+To stop the processing of workflow commands, pass a unique token to `stop-commands`. To resume processing workflow commands, pass the same token that you used to stop workflow commands.
```

Comment:
👋 Hi @fhammerl -- I've added this sentence to further explain how the value is used. What do you think of this?